save-cosv/src/main/kotlin/com/saveourtool/save/cosv/controllers/CvssController.kt

```diff
@@ -0,0 +1,38 @@
+package com.saveourtool.save.cosv.controllers
+
+import com.saveourtool.save.configs.ApiSwaggerSupport
+import com.saveourtool.save.cosv.cvsscalculator.scoreCalculator
+import com.saveourtool.save.utils.StringResponse
+import com.saveourtool.save.utils.blockingToMono
+import com.saveourtool.save.v1
+
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import reactor.core.publisher.Mono
+
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+/**
+ * Rest controller for CVSSs
+ */
+@ApiSwaggerSupport
+@RestController
+@RequestMapping("/api/$v1/cvss")
+class CvssController {
+    /**
+     * @param vector
+     * @return base score criticality
+     */
+    @GetMapping("/get-base-score")
```

Comment:
do we really need it as a rest endpoint?

Reply:
I want to make a full-fledged calculator for calculations and show users the calculated value

Reply:
Then we can transfer this logic to a separate module

Reply:
I understand, but it's a just multiplying of several numbers. Do we really need to make a request to backend for such calculation?

Reply:
Do you propose to move this logic to the common or to the frontend?

Reply:
to the common

Reply:
Ok. Moved it